src/libraries/System.Private.CoreLib/src/System/Environment.Unix.cs

```diff
@@ -26,6 +26,12 @@ public static string MachineName
 
         public static string UserName => Interop.Sys.GetUserNameFromPasswd(Interop.Sys.GetEUid());
 
+        private static bool IsAdminProcess()
+        {
+            uint userId = Interop.Sys.GetEUid();
+            return (userId == 0);
+        }
```

Comment:
For consistency with the other methods around it, and for conciseness:
```suggestion
        private static bool IsAdminProcess() => Interop.Sys.GetEUid() == 0;
```